ddsrouter_event/include/ddsrouter_event/SignalManager.hpp

```diff
@@ -176,10 +176,10 @@ class SignalManager
     std::atomic<bool> signal_handler_thread_stop_;
 
     //! Counter incremented when a signal arrives and decremented after being read by \c signal_handler_thread_
-    std::atomic<uint32_t> signals_received_;
+    static std::atomic<uint32_t> signals_received_;
```

Comment:
NIT
Please, move every static variable together

Reply:
Attributes are grouped by functionality, I have anyway grouped static methods and variables within each subgroup.